src/popups/HandicapPopup/HandicapPopup.js

```diff
@@ -61,10 +61,13 @@ function HandicapPopup({ feature }) {
           ? `${t('behig_nach')} 2026`
           : properties.prognose;
       return `${t(
-        'Dieser Bahnhof wird barrierefrei umgebaut (Prognose)',
-      )}: ${prognose}.`;
+        `Dieser Bahnhof wird vollständig angepasst`,
+      )}: ${prognose} (${t('Prognose')}).`;
     }
-    return t('Dieser Bahnhof wird nicht baulich angepasst.');
+    if (status === 'NICHT OK') {
```

Comment:
Is it really `NICHT OK`? In the behig table in the database `NICHT OK` does not exist but `BLEIBEN NICHT OK`.

Reply:
That's right! Good catch :)